plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/edit.tsx

```diff
@@ -55,12 +57,25 @@ const INNER_BLOCKS_TEMPLATE: InnerBlockTemplate[] = [
 	],
 ];
 
-// Get product type options.
-const productTypes = getProductTypeOptions();
-
 const AddToCartOptionsEdit = ( props: BlockEditProps< Attributes > ) => {
 	const { setAttributes } = props;
 
+	const { productTypes, currentProduct } = useSelect< {
```

Comment:
Should the `currentProduct` variable be named `currentProductType` instead?

Reply:
addressing here